main.py

```diff
@@ -2,6 +2,35 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+
+    genres = [
+        "Western", "Action", "Dramma"
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [
+        "George Klooney", "Kianu Reaves",
+        "Scarlett Keegan", "Will Smith",
+        "Jaden Smith", "Scarlett Johansson"
+    ]
```

Comment:
You don't need to use a list when you can have a tuple it is more optimized if you don't change data